babel/src/main/java/org/apache/calcite/sql/babel/SqlTranslateUsingCharacterSet.java

```diff
@@ -29,19 +29,30 @@
 
 import java.util.List;
 
+/**
+ * A <code>SqlTranslateUsingCharacterSet</code> is an AST node that describe
+ * the Translate Using CharacterSet to CharacterSet with Error
+ */
 public class SqlTranslateUsingCharacterSet extends SqlCall implements SqlExecutableStatement {
-  public static final SqlSpecialOperator OPERATOR =
+  private static final SqlSpecialOperator OPERATOR =
       new SqlSpecialOperator("TRANSLATE ", SqlKind.OTHER_FUNCTION);
 
-  final List<SqlNode> args;
-  final boolean isWithError;
+  private final List<SqlNode> args;
+  private final boolean isWithError;
 
+  /**
+   * Creates a {@code SqlTranslateUsingCharacterSet}.
+   *
+   * @param pos  Parser position, must not be null
+   * @param args  it contains two SqlNode, the first one is an char sequence to be
+   *              translated, the second one is the AST node of SqlCharacterSetToCharacterSet
+   * @param isWithError if the with error token is specified
```

Comment:
nit: Uppercase "i"

Reply:
Fix in https://github.com/googleinterns/calcite/pull/84/commits/ecfca9bab8c080bbfd97f37bc456fdd3212aa004